Source/WebInspectorUI/UserInterface/Base/Main.js

```diff
@@ -290,6 +290,8 @@ WI.contentLoaded = function()
     WI.detailsSidebar.addEventListener(WI.Sidebar.Event.WidthDidChange, WI._sidebarWidthDidChange, WI);
     WI.detailsSidebar.addEventListener(WI.Sidebar.Event.CollapsedStateDidChange, WI._sidebarWidthDidChange, WI);
     WI.detailsSidebar.addEventListener(WI.MultiSidebar.Event.MultipleSidebarsVisibleChanged, WI._sidebarWidthDidChange, WI);
+    WI.detailsSidebar.addEventListener(WI.MultiSidebar.Event.BottomnessDidChange, WI._sidebarWidthDidChange, WI);
+    WI.detailsSidebar.addEventListener(WI.MultiSidebar.Event.HeightDidChange, WI._sidebarWidthDidChange, WI);
```

Comment:
Please don't reuse handlers for different events. A new `_sidebarHeightDidChange` handler would be needed.

Reply:
I've done the latter.